libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_TCP_WIN.c

```diff
@@ -1810,15 +1862,15 @@ const int32_t l500ms = 500;
 			if( pxSegment->u.bits.bOutstanding != pdFALSE_UNSIGNED )
 			{
 				/* As 'ucTransmitCount' has a minimum of 1, take 2 * RTT */
-				ulMaxTime = ( ( uint32_t ) 1u << pxSegment->u.bits.ucTransmitCount ) * ( ( uint32_t ) pxWindow->lSRTT );
+				ulMaxTime = ( ( uint32_t ) 1U << pxSegment->u.bits.ucTransmitCount ) * ( ( uint32_t ) pxWindow->lSRTT );
 
 				if( ulTimerGetAge( &( pxSegment->xTransmitTimer ) ) < ulMaxTime )
 				{
-					ulLength = 0ul;
+					ulLength = 0UL;
 				}
 			}
 
-			if( ulLength != 0ul )
+			if( ulLength != 0UL )
```

Comment:
Sorry for adding this coment here but I cannot add my comment on line 1891. It should be `BaseType_t xTCPWindowTxDone( const TCPWindow_t *pxWindow )` according to the new declaration in FreeRTOS_TCP_WIN.h

Reply:
This is updated just now. Thanks